src/openrct2/management/NewsItem.cpp

```diff
@@ -484,3 +484,23 @@ void News::RemoveItem(int32_t index)
     }
     gameState.NewsItems[newsBoundary - 1].Type = News::ItemType::Null;
 }
+
+void News::importNewsItems(
+    GameState_t& gameState, const std::vector<News::Item>& recent, const std::vector<News::Item>& archived)
```

Comment:
The implementation being a vector isn't really relevant here, right. I'd suggest taking the C++20 approach of using std::span for the parameters. (The construction is implicit on the calling side.)